components/markets/market-card/index.tsx

```diff
@@ -241,14 +225,29 @@ const MarketCard = ({
     <MarketCardContext.Provider value={{ baseAsset }}>
       <div
         data-testid={`marketCard-${marketId}`}
-        className={`group flex flex-col min-w-full md:min-w-[calc(50%-14px)] lg:min-w-[calc(100%/3-18.67px)] h-[171px] 
-        rounded-[10px] p-5 relative bg-white hover:bg-pastel-blue ${className}`}
+        className={`group flex flex-col min-w-full md:min-w-[calc(50%-14px)] lg:min-w-[calc(100%/3-18.67px)]  
+        rounded-[10px] p-5 relative bg-white ztg-transition hover:scale-105 ${className}`}
       >
         <Link
           href={`/markets/${marketId}`}
-          className="flex flex-col flex-1 gap-4"
+          onClick={(e) => {
+            if (disableLink) {
+              e.preventDefault();
+              return;
+            }
+          }}
+          className={`flex flex-col flex-1 gap-4 ${
```

Comment:
Yeah that error cant easily be removed since the client will actually render a bit different from the server, it was related to the issue I was talking about earlier. See this bit of code: https://github.com/zeitgeistpm/ui/pull/1737/files#diff-8a23978fb2a961548d1b3b3c2a25caa7d32c079a3411b806ee395a86288d007aR109-R132